runtimes/standalone/src/lib.rs

```diff
@@ -283,8 +282,8 @@ impl pallet_transaction_payment::Config for Runtime {
 }
 
 impl pallet_sudo::Config for Runtime {
-	type Event = Event;
-	type Call = Call;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
```

Comment:
The PR description says that the convetion seems to be to keep using `Call`, `Event`, `Call` etc for pallets, but I see here that that is not really the case. I guess we should also introduce the change in our pallets.

Reply:
You are right. Will fix that!

Reply:
fde113cb1b81146770db9d3edfab89e40100617a